packages/grafana-ui/src/components/BarGauge/BarGauge.tsx

```diff
@@ -111,25 +112,37 @@ export class BarGauge extends PureComponent<Props> {
   }
 
   renderBasicAndGradientBars(): ReactNode {
-    const { value, showUnfilled } = this.props;
+    const { value, showUnfilled, valueMode: valueDisplayMode } = this.props;
```

Comment:
Why are we renaming the `valueMode` here? It seems to be renamed only in a couple places.